src/handlers/handlerUtils.ts

```diff
@@ -1053,6 +1054,10 @@ export function constructConfigFromRequestHeaders(
     openaiProject: requestHeaders[`x-${POWERED_BY}-openai-project`],
   };
 
+  const martianConfig = {
```

Comment:
Hey! I can see that this mapping was added in the PR. But its not used anywhere in the provider config. Can you please verify if this is required? We can remove it if its not used anywhere